src/back_end/docs/Events.md

```diff
@@ -0,0 +1,53 @@
+## <span style="color:deepskyblue">Events Data Endpoints</span>
+Documentation for all callable back-end functions related to Events data
+
+## <span style="color:deepskyblue"> getEventsBasedOnTime
+Gets events based on the current date/time. Returns the output as a list, sorted in ascending order by date (closest to today first)
+
+* **Function Call:** getEventsBasedOnTime(upcoming, verbose)
+
+* **Params:**
+
+    | Name       | Type    | Description                                               | Default                 |
+    | ---------- | ------- | --------------------------------------------------------- | ----------------------- |
+    | `upcoming` | Boolean | Indicates querying for upcoming events (Alternative: old) | NONE (Required Param)   |
+    | `verbose`  | Boolean | Indicates returning all data (Alternative: just the name) | NONE (Required Param)   |
+
+* **Returned Data Format:** List
+
+* **Success Response: (verbose)** <br>
+Sample response shows output format, followed by sample data.
+  ```
+  [
+    {
+      key: <String>,
+      value: {
```

Comment:
Instead of having a "key" and a "value", is it possible to just make the name of the event a field in the value? For example, make the output be:
```
[
       {
          Name: "Event name"
          Attendees: 21,
          Date: '2192-10-20T16:00-07:00',
          Description: 'This is an event in the future',
          Image: 'blob (do not use, not yet implemented)',
          Location: 'UW?',
          Sponsor: 'Tettie'
      },
      {...},
      {...},
]
```

Reply:
Yeah I could do that, the original idea was to keep the title separate for output purposes (e.g. for Verbose, it would be)
```
[
  {
     key: <name>
     value: <obj>
  }
]
```
And for non-verbose,
```
[
  {
     key: <name>
     value: undefined
  }
]
```
But I guess we can do it like above too

Reply:
@carneyb3592 ^ Think that also answers your question

Reply:
@sidlak-c137 do you still want this change? Or should I change the verbose output?

Reply:
Then yeah verbose is better, I was just hoping there was a way to just return an array of values lmao